imports/plugins/included/taxes-taxcloud/client/components/taxCloudSettingsForm.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Form } from "/imports/plugins/core/ui/client/components";
+import { TaxCloudPackageConfig } from "../../lib/collections/schemas";
+
+/**
+ * @file TaxCloudSettingsForm is a React Component used to change TaxCloud
+ * settings.
+ * @module TaxCloudSettingsForm
+ */
+
+/**
+ * @method TaxCloudSettingsForm
+ * @summary renders a form for updating TaxCloud settings.
+ * @param {Object} props - some data for use by this component.
+ * @property {Function} handleSubmit - a function for saving new TaxCloud settings.
+ * @property {Array} hiddenFields - the fields (of the TaxCloud Package) to hide from the form.
+ * @property {Object} settings - the value of the "settings" field in the TaxCloud Package.
+ * @property {Object} shownFields - info about the fields the form is to show.
+ * @since 1.5.2
+ * @return {Node} - a React node containing the TaxCloud settings form.
+ */
+const TaxCloudSettingsForm = (props) => {
+  const { handleSubmit, hiddenFields, settings, shownFields } = props;
+  return (
+    <div className="rui taxcloud-settings-form">
+      {!settings.taxcloud.apiLoginId &&
+        <div className="alert alert-info">
+          <span data-i18n="admin.taxSettings.taxcloudCredentials">Add API Login ID to enable</span>
```

Comment:
Don't use `data-i18n` use the translation component `<Components.Translation defaultValue="" i18nkey="" />`